sonoff/xdrv_90_deepsleep.ino

```diff
@@ -0,0 +1,170 @@
+/*
+  xdrv_90_deepsleep.ino - Shutter/Blind support for Sonoff-Tasmota

```

Comment:
"Deep sleep support for Sonoff-Tasmota" ?

Reply:
”Shutter / Blind is wrong”. Copy & Paste error. Yes deepsleep for battery based esp8266 like mentioned in the push request.